models/contacts.js

```diff
@@ -1,19 +1,91 @@
-// const fs = require('fs/promises')
+const path = require("path");
+const fs = require("fs/promises");
+const { v4: uuidv4 } = require("uuid");
+const Joi = require("joi");
 
-const listContacts = async () => {}
+const contactsPath = path.resolve("models", "contacts.json");
 
-const getContactById = async (contactId) => {}
+const listContacts = async () => {
+  try {
+    const contactsJSON = await fs.readFile(contactsPath);
+    const contacts = JSON.parse(contactsJSON);
+    return contacts;
+  } catch {
+    console.log("error when reading the file");
```

Comment:
you need to handle `catch` somehow. What will be returned in your `api`for this particular case?